src/test/java/teammates/test/cases/ui/InstructorFeedbackResultsDownloadActionTest.java

```diff
@@ -113,8 +119,45 @@ public void testExecuteAndPostProcess() throws Exception {
             assertEquals(String.format(Const.StatusCodes.NULL_POST_PARAMETER,
                     Const.ParamsNames.FEEDBACK_SESSION_NAME), e.getMessage());
         }
+        
+        ______TS("Typical case: results with a filter text");
+
+        action = getAction(paramsWithFilterText);
+        result = (FileDownloadResult) action.executeAndPostProcess();
+        expectedDestination = "filedownload?" + "error=false" + "&user=idOfInstructor1OfCourse1";
+        assertEquals(expectedDestination, result.getDestinationWithParams());
+        assertFalse(result.isError);
+
+        expectedFileName = session.getCourseId() + "_" + session.getFeedbackSessionName();
+        assertEquals(expectedFileName, result.getFileName());
+        verifyFileContentForDownloadWithFilterText(result.getFileContent(), session);
     }
 
+    private void verifyFileContentForDownloadWithFilterText(String fileContent,
+            FeedbackSessionAttributes session) {
+        /*
+        full testing of file content is
+        in FeedbackSessionsLogicTest.testGetFeedbackSessionResultsSummaryAsCsv()
+        */
+        
+        String[] expected = {
+        // CHECKSTYLE.OFF:LineLength csv lines can exceed character limit
+                "Course,\"" + session.getCourseId() + "\"",
+                "Session Name,\"" + session.getFeedbackSessionName() + "\"",
+                "",
+                "",
+                "Question 3,\"My comments on the class\"",
+                "",
+                "Team,Giver's Full Name,Giver's Last Name,Giver's Email,Recipient's Team,Recipient's Full Name,Recipient's Last Name,Recipient's Email,Feedback",
```

Comment:
This one is fine. Leave CSV lines as is as indicated by the suppression.
But you may want to indent the comment in the same level as `"Course,\""`